homeassistant/components/zwave_js/const.py

```diff
@@ -33,3 +33,10 @@
 ATTR_PROPERTY_NAME = "property_name"
 ATTR_PROPERTY_KEY_NAME = "property_key_name"
 ATTR_PARAMETERS = "parameters"
+
+# service constants
+SERVICE_SET_CONFIG_VALUE = "set_config_value"
+
+ATTR_CONFIG_PROPERTY = "property"
```

Comment:
```suggestion
ATTR_CONFIG_PROPERTY = "parameter"
```
Also consistent with other integrations, and more familiar terminology for users.

Reply:
parameter or config_label ?